source/wp-content/themes/wporg-developer-2023/patterns/front-page-content.php

```diff
@@ -232,7 +232,7 @@
 
 <!-- wp:column {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}}} -->
 <div class="wp-block-column"><!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"700","lineHeight":1.6},"elements":{"link":{"color":{"text":"var:preset|color|blueberry-2"}}}},"textColor":"blueberry-2","className":"is-style-short-text","fontSize":"normal","fontFamily":"inter"} -->
-<h3 class="wp-block-heading is-style-short-text has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700;line-height:1.6"><a href="https://learn.wordpress.org"><?php esc_html_e( 'Local Development Environment', 'wporg' ); ?></a></h3>
+<h3 class="wp-block-heading is-style-short-text has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700;line-height:1.6"><a href="https://developer.wordpress.org/themes/getting-started/setting-up-a-development-environment/"><?php esc_html_e( 'Local Development Environment', 'wporg' ); ?></a></h3>
```

Comment:
On the current homepage this links to https://make.wordpress.org/core/handbook/installing-a-local-server/

Intentional change?

Reply:
To be honest I didn't realize these were existing links and searched for the most relevant ones via our global search. I'll update. :)

Reply:
Updated: https://github.com/WordPress/wporg-developer/pull/378/commits/0fb4f9b762285e6b72249538e1c45c41d9528910